common/flatpak-installation.c

```diff
@@ -1519,6 +1519,40 @@ flatpak_installation_get_config (FlatpakInstallation *self,
   return flatpak_dir_get_config (dir, key, error);
 }
 
+/**
+ * flatpak_installation_get_min_free_space_bytes:
+ * @self: a #FlatpakInstallation
+ * @out_bytes: (out): Location to store the result
+ * @error: Return location for a #GError
+ *
+ * A convenience wrapper around ostree_repo_get_min_free_space_bytes().
+ * This can be used to query the min-free-space-* config option of the
+ * repo by clients like gnome-software.
+ *
+ * Returns: %TRUE on success, or %FALSE on error.
+ * Since: 1.1.0
+ */
```

Comment:
In other places, I have just used

> Since:

We should be consistent with this

 1.1